src/app/reporting/Engine.cpp

```diff
@@ -204,16 +204,16 @@ CHIP_ERROR Engine::BuildSingleReportDataAttributeReportIBs(ReportDataMessage::Bu
                                       pathForRetrieval, &encodeState);
             if (err != CHIP_NO_ERROR)
             {
-                ChipLogError(DataManagement,
-                             "Error retrieving data from clusterId: " ChipLogFormatMEI ", err = %" CHIP_ERROR_FORMAT,
-                             ChipLogValueMEI(pathForRetrieval.mClusterId), err.Format());
-
                 // If error is not an "out of writer space" error, rollback and encode status.
                 // Otherwise, if partial data allowed, save the encode state.
                 // Otherwise roll back. If we have already encoded some chunks, we are done; otherwise encode status.
 
                 if (encodeState.AllowPartialData() && IsOutOfWriterSpaceError(err))
                 {
+                    ChipLogDetail(DataManagement,
+                                  "Encoding is out of space, and partial data allowed, save the encode state for report chunking "
```

Comment:
Might be clearer as:
```suggestion
                                  "List does not fit in packet, chunk between list items for clusterId: " ....
```